quilt_lite/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "quilt_lite"
+version = "0.1.0"
+description = ""
+authors = ["Alexei Mochalov <alexei@quiltdata.io>"]
+
+[tool.poetry.dependencies]
+# TODO: make it working under 3.7
+python = "^3.8"
+ariadne = {git = "https://github.com/mirumee/ariadne.git"}
```

Comment:
Any reason we don't use released version here?

Reply:
ping

Reply:
yep, there was some reason, but i don't remember exactly, there were some dependency version conflicts or smth like this. i see the next version has been released since then, so it's probably ok to use it (0.14)